process/project-lifecycle.md

```diff
@@ -1,156 +1,168 @@
 
-#### Gives - Review and Reporting
-*** Need Text ***
+#### Project Graduation Process: Incubating to Graduation
 
-#### Gets - Benefits
-*** Need Text ***
+Graduation requires undergoing due diligence as a part of the process to move from Incubation to Graduation. Due Diligence is driven by a TAC or parent WG sponsor. Once the diligence is complete and the proposal made, two weeks are allocated for public comment before a TAC vote is called.
 
-#### Project Graduation Process: Graduated to Archiving
+#### Submission Forms for Graduation
 
-Open source projects have a lifecycle and there are times that projects become inactive due to a variety of reasons. There are also cases where a project may no longer want to be supported by the TAC, or the TAC may no longer wish to recommend the use of a project.
+1. [Proposal](TBD)
+2. [Legal](TBD)
 
-#### Archiving Criteria
 
-When voting on a proposal to archive a project, TAC members may wish to consider whether the project continues to meet the criteria for OpenSSF acceptance. The TAC may also look at activity levels in the project (https://all.devstats.cncf.io/d/53/projects-health-table?orgId=1), although it is important to note that there is a difference between a mature project that doesn't get much attention any more but is stable, versus a project that is inactive.
+### Archiving
 
-#### Submission Forms for Graduated to Archived
+Open source projects have a lifecycle and there are times that projects become inactive due to a variety of reasons. There are also cases where a project may no longer want to be supported by the OpenSSF, or the OpenSSF TAC may no longer wish to recommend the use of a project. Archiving happens through a vote of the TAC, and can be requested by the corresponding project's lead(s) or a TAC member.
 
-1. [Proposal](TBD)
-2. [Legal](TBD)
+#### Archiving Considerations
 
-### Archived
+When voting on a proposal to archive a project, TAC members may consider:
+* If the project has consistently met the Project Requirements
+* If the project still supports the OpenSSF mission
+* If the project's inactivity or inconsistent maintenance presents a user security risk
+* If the project design or approach is no longer a recommended best practice
 
-Archived projects are no longer in active development and are only archived after a TAC or parent WG vote.
+#### Archive Process
+
+Archived projects are no longer in active development and are only archived after a TAC vote.
 
 What does archiving for an OpenSSF project mean?
-* OpenSSF will no longer provide support for the project via service desk
-* OpenSSF will list archived projects online
-* Trademarks and domain names of archived projects are still hosted by the OpenSSF and the Linux Foundation
+* OpenSSF will no longer provide budgetary, event, or marketing support for the project
+* Project maintainers may request trademark transfers from the OpenSSF and the Linux Foundation
```

Comment:
"may request" is, IMHO, appropriately open-ended language, and imposes no obligation upon the LF. 

However, I think the previous line should remain in the doc.

Reply:
I agree with @AevaOnline though. I think we should have both:
 * Trademarks and domain names of archived projects are still hosted by the OpenSSF and the Linux Foundation 
 * Project maintainers may request trademark transfers from the OpenSSF and the Linux Foundation
So that if no transfer is requested (and approved) the trademarks aren't just dropped.

Reply:
+1 for option to ask, but no obligation to release the mark.

Reply:
@lehors good text clarification -- my edit didn't mean to imply they were auto-released. I'll edit.